exercises/sieve/instructions.md

```diff
@@ -5,24 +5,36 @@ Your task is to create a program that implements the Sieve of Eratosthenes algor
 A prime number is a number that is only divisible by 1 and itself.
 For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
```

Comment:
I like negative examples.

```suggestion
A prime number is a number that is only divisible by 1 and itself.
For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
In contrast, 6 is not a prime number because it is divisible not only by 1 and itself, but also by 2 and 3.
```

Reply:
I would be tempted to say:
For example, 2, 3, 5, 7, 11, and 13 are the first six prime numbers.